buildOnPrem.sh

```diff
@@ -0,0 +1,3 @@
+yarn onprem:prep \
+&& yes | vsce package \
```

Comment:
Make this file a yarn script?

Reply:
can't because it is modifying the package.json and that didn't work